### PR TITLE
added double-struck cool dictionary

### PR DESCRIPTION
**Pull request by nayonika1999**

### CODE REVIEW

**Comment by nayonika1999** [comment]
fixes #3

**Comment by nayonika1999** [comment]
Resolved conflicts.Please have a look.

File: services/coolifyService.js [hunk 1]
```diff
@@ -245,6 +245,34 @@ coolifyDicts.wavy = {
 	'y':'Ɏ',
 	'z':'Ⱬ',
   }
+  coolifyDicts.doublestruck={
```
Comment (mohitkyadav) [inline services/coolifyService.js]: Insert a blank line after line 247

File: services/coolifyService.spec.js [hunk 2]
```diff
@@ -59,5 +59,9 @@ describe('coolifyService.alphaNumericeName', () => {
 
   it('should replace with dot letter', () => {
   expect(alphaNumericeName('hello')[12]).to.equal('ȟ£ȽȽ¤');
+    
+  it('should replace with doublestruck letter', () => {
+  expect(alphaNumericeName('hello')[13]).to.equal('𝕙𝕖𝕝𝕝𝕠');
```
Comment (mohitkyadav) [inline services/coolifyService.spec.js]: Please indent properly.

File: services/uncoolifyService.js [hunk 3]
```diff
@@ -252,6 +252,34 @@ uncoolifyDicts.curr = {
   'Ɏ': 'y',
   'Ⱬ': 'z'
 };
+uncoolifyDicts.doublestruck = {
```
Comment (mohitkyadav) [inline services/uncoolifyService.js]: Insert blank line after 254.

File: services/uncoolifyService.spec.js [hunk 4]
```diff
@@ -70,5 +70,8 @@ describe('uncoolifyService.alphaNumericeName', () => {
 
   it('should uncoolify dot letters', () => {
     expect(alphaNumericeName('ȟ£ȽȽ¤')[0]).to.equal('hello');
+    
+ it('should uncoolify doublestruck letters', () => {
```
Comment (mohitkyadav) [inline services/uncoolifyService.spec.js]: Incorrect indentation.

**Comment by nayonika1999** [comment]
@mohitkyadav  I made the changes locally and then committed and rebased origin. Are the changes reflected in this PR or do i have to make a new pr again?

File: services/coolifyService.js [hunk 5]
```diff
@@ -358,7 +359,7 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_curr = name.allReplace(coolifyDicts.curr);
 		let cool_name_fairy = name.allReplace(coolifyDicts.fairy);
 		let cool_name_dot = name.allReplace(coolifyDicts.dot);
-    let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
+    	let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
```
Comment (mohitkyadav) [inline services/coolifyService.js]: Wrong indentation.

File: server.js [hunk 6]
```diff
@@ -37,7 +37,7 @@ app.get("/coolify", (request, response) => {
 		cool_name_curr: cool_names[10],
 		cool_name_fairy: cool_names[11],
 		cool_name_dot: cool_names[12],
-    cool_name_doublestruck: cool_names[13],
+   	    cool_name_doublestruck: cool_names[13],
```
Comment (mohitkyadav) [inline server.js]: Wrong indentation

**Comment by nayonika1999** [comment]
I hope i fixed the indentation errors.
Kindly review.

**Pull request merged**